README.md

```diff
@@ -89,17 +89,255 @@ Name of the application pool.
 
 The state of the ApplicationPool. Must be either `Started` or `Stopped`
 
+#### `auto_start`
+
+When true, indicates to the World Wide Web Publishing Service (W3SVC) that the application pool should be automatically started when it is created or when IIS is started. Valid options `true` or `false`
+
+#### `clr_config_file`
+
+Specifies the .NET configuration file for the application pool
+
+#### `enable32_bit_app_on_win64`
+
+When true, enables a 32-bit application to run on a computer that runs a 64-bit version of Windows. Valid options `true` or `false`
+
+#### `enable_configuration_override`
+
+When true, indicates that delegated settings in Web.config files will processed for applications within this application pool. When false, all settings in Web.config files will be ignored for this application pool. Valid options `true` or `false`
+
 #### `managed_pipeline_mode`
 
-The managedpipelinemode of the ApplicationPool. Must be either `Integrated` or `Classic`
+Specifies the request-processing mode that is used to process requests for managed content. Must be either `Integrated` or `Classic`
+
+#### `managed_runtime_loader`
+
+Specifies the managed loader to use for pre-loading the the application pool.Note: This property was added in IIS 7.5.The default value is `webengine4.dll`
```

Comment:
String values should be in single quotes; needs a period at the end. Needs space between sentences.